.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,52 @@
+name: Feature Request
+description: Suggest a new idea for this project
+labels: ["Type: Feature Request", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for showing interest in adding new features to Open Library! 👍
+
+  - type: textarea
+    attributes:
+      label: Problem
+      description: Describe the problem that this feature will address.
+      value: |
+        ### A clear and concise description of what you want to happen
+
+        ### Expected behaviour / screenshots (ex: Figma design screenshots for UI feature)
+
+    validations:
+      required: true
+
```

```suggestion
        ### Expected behaviour / screenshots (ex: Figma design screenshots for UI feature)

```
Remove validations, it is not required.